service.go

```diff
@@ -68,10 +62,11 @@ func (s *Service) Run(ctx context.Context) error {
 }
 
 func writeErrorReply(w io.Writer, err error) error {
-	responseNotOk := birch.EC.Int32("ok", 0)
-	errorDoc := birch.EC.String("errmsg", err.Error())
-	doc := birch.NewDocument(responseNotOk, errorDoc)
-
+	responseNotOK := birch.EC.Int32("ok", 0)
```

Comment:
shouldn't this be 1 if err == nil?

Reply:
The errmsg is entirely optional. ok 0 means that either there was an error (errmsg) or operation is not supported/not relevant (not considered an error).